src/LoginServer/Network/LoginPacketSender.cs

```diff
@@ -67,6 +68,13 @@ public static void BC_COMMANDER_LIST(LoginConnection conn)
 				for (int i = 0; i < Items.EquipSlotCount; ++i)
 					packet.PutShort(0);
 
+				// Unknown
+				packet.PutByte(1);
+				packet.PutByte(1);
+				packet.PutByte(1);
+				packet.PutByte(1);
+				packet.PutByte(0);
```

Comment:
These bytes do not appear to be associated with the equipment properties structure.